src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring/containerized-images.mdx

```diff
@@ -35,21 +35,43 @@ New Relic offers pre-built images that include our layer's different runtimes: J
     * [Java](https://gallery.ecr.aws/newrelic-lambda-layers-for-docker/newrelic-lambda-layers-java)
     * [Node.js](https://gallery.ecr.aws/newrelic-lambda-layers-for-docker/newrelic-lambda-layers-nodejs) 
     * [Python](https://gallery.ecr.aws/x6n7b2o2/newrelic-lambda-layers-python)
+    * [Ruby](https://gallery.ecr.aws/x6n7b2o2/newrelic-lambda-layers-ruby)
     * [Extension-only](https://gallery.ecr.aws/newrelic-lambda-layers-for-docker/newrelic-lambda-layers-lambdaextension)
-1. At the top of your Dockerfile, add the following code: 
+1. In your multi-stage Dockerfile, use New Relic's prebuilt image as an intermediate build stage. Then, copy New Relic's agent and extension from our prebuilt image into your final build stage.
+
+### Example
```

Comment:
@homelessbirds I didn't notice this before, but this Node.js example is different from what's in the tabs below. Why don't the examples in the tabs show the base image?

Reply:
@rhetoric101 the base image is dependent on what the user chooses. We can't pick it for them.

Reply:
Also, those are just examples-- they might be different for each person, but they just show what it could look like based on the most popular use case